cmd/releasego/testdata/update-azure-linux/pr/pr-description-essential.golden.md

```diff
@@ -0,0 +1,14 @@
+golang: bump Go version to 1.23.0-2
+
+---
+
+Hi! 👋 I'm the Microsoft Go team's bot. This is an automated pull request I generated to bump the Go version to [1.23.0-2](https://github.com/microsoft/go/releases/tag/v1.23.0-2).
+
+I'm not able to run the Azure Linux pipelines yet, so the Microsoft Go release runner will need to finalize this PR. @a-go-developer
+
+Finalization steps:
+- Trigger [Source Tarball Publishing](https://dev.azure.com/mariner-org/mariner/_build?definitionId=2284) with Full Name `go1.23.0-20240708.2.src.tar.gz` and URL `https://github.com/microsoft/go/releases/download/v1.23.0-2/go1.23.0-20240708.2.src.tar.gz`
```

Comment:
Does Azure Devops allow us to pass job parameters as URL params? I'm wondering if we can make a 1-click link that's correctly configured?

Reply:
Unfortunately not. Can't even get the Run dialog to be open. ChatGPT and Copilot came up with compelling hallucinations, though.